extension-config/popup.js

```diff
@@ -0,0 +1,63 @@
+function getActiveTab() {
```

Comment:
Can you add a comment to the top of the file explaining the purpose of this code?